templates/v2/vault/contact/_card.html.twig

```diff
@@ -1,5 +1,6 @@
-<tr class="text-primary text-center align-middle">
-    <td class="bg-white borderless p-0">
+<tr class="text-primary text-center align-middle" {{ stimulus_controller('ajax-toggle', {'url': path('contact_toggle_visibility', {'id': contact.id}) })}}>
+    <td class="bg-white borderless p-0 personal-data-card {{ contact.bPrive ? 'private' : 'shared' }} position-relative"{{ stimulus_target('ajax-toggle', 'card') }}
+        data-toggle-classes="shared private">
```

Comment:
```
<td class="bg-white borderless p-0 personal-data-card position-relative {{ contact.bPrive ? 'private' : 'shared' }}"
       {{ stimulus_target('ajax-toggle', 'card') }}
        data-toggle-classes="shared private">
```
I think that having conditional data class (`{{ contact.bPrive ? 'private' : 'shared' }}`) at the end of the class list and an end of line before the stimulus target attribute make it easier to read. 
There are several occurrence in the other templates where you could apply this :)

Reply:
Done